servers/dsub/jobs/controllers/utils/extensions.py

```diff
@@ -0,0 +1,19 @@
+from jobs.controllers.utils import logs
+from jobs.models.extended_fields import ExtendedFields
+
+
+def get_extensions(job):
+    """Extracts ExtendedFields from a job, if present.
+
+        Args:
+            job: A dict with dsub job metadata
+
+        Returns:
+            ExtendedFields: Populated extensions on the job
+    """
+    return ExtendedFields(
+        user_id=job['user-id'],
+        status_detail=job['status-detail'],
```

Comment:
Are `status-detail` and `last-update` sure things? Will they ever be `None`? Maybe safer to just use `get()` instead

Reply:
Good point, changed to `get()`